nativeauthenticator/handlers.py

```diff
@@ -83,10 +85,10 @@ async def post(self):
             'email': self.get_body_argument('email', '', strip=False),
             'has_2fa': bool(self.get_body_argument('2fa', '', strip=False)),
         }
+        taken = self.authenticator.user_exists(user_info['username'])
         user = self.authenticator.create_user(**user_info)
-        name = self.authenticator.user_exists(user_info['username'])
```

Comment:
Variable name was confusing, this is a boolean that indicates if the name is taken, not the name as a string.